nova/virt/lxd/driver.py

```diff
@@ -1325,8 +1325,8 @@ def _after_reboot(self):
     def _migrate(self, source_host, instance):
         """Migrate an instance from source."""
         source_client = pylxd.Client(
-            endpoint='https://{}'.format(source_host), verify=False)
+                endpoint='https://{}:8443'.format(source_host), verify=False)
```

Comment:
Not sure if it matters, but the indentation appears to have changed on this line.